Apps/WebClient/src/ClientApp/src/views/reports.vue

```diff
@@ -36,10 +35,15 @@ Vue.component("BFormTag", BFormTag);
     },
 })
 export default class ReportsView extends Vue {
+    @Action("retrieve", { namespace: "patient" })
```

Comment:
I have used this opportunity to refactor the reports which is not required for this task/story, but I can roll these changes back if needed.

Reply:
+1 For refactoring!